rules/data/embedded/embedded-svg.yara

```diff
@@ -0,0 +1,46 @@
+rule svg: harmless {
+  meta:
+    description = "Contains SVG (Scalable Vector Graphics) content"
+
+  strings:
+    $svg_open1  = "<svg>"
+    $svg_open2  = "<SVG>"
+    $svg_close1 = "</svg>"
+    $svg_close2 = "</SVG>"
+
+  condition:
+    all of them
+}
+
+rule foreign_object: high {
```

Comment:
reading through https://developer.mozilla.org/en-US/docs/Web/SVG/Element/foreignObject - I think this should be a "medium".

Reply:
Updated in [`19224b3` (#704)](https://github.com/chainguard-dev/malcontent/pull/704/commits/19224b38d27b23d94c32d35dda2d5b3bebfe0121).